src/main/java/com/group12/ciserver/controller/EventController.java

```diff
@@ -16,4 +24,37 @@
         log.info("Received push event, pushEvent={}", pushEvent);
         return ResponseEntity.noContent().build();
     }
+
+    @GetMapping("/history")
+    @ResponseBody
+    public ArrayList<BuildInfo> buildHistory() {
+
+        DatabaseWrapper db = new DatabaseWrapper();
+
+        return db.getAllBuilds();
+    }
+
+    @GetMapping("/history/{buildId}")
+    @ResponseBody
+    public ResponseEntity<BuildInfo> buildInfo(@PathVariable String buildId) {
```

Comment:
Is it possible to use `Long` (class, not primitive) instead of `String` for the `buildId`? You then don't have to convert it to a `long` yourself.

Reply:
I will look into it. Right now, if it fails to convert it should return 400 bad request.

Reply:
Seems like it would add complexity. I'm not finding a very straightforward way.

Reply:
Sure you can keep it like that.